docs/deploy_docs.md

```diff
@@ -2,13 +2,6 @@
 
 Each token to be bridged needs to have a corresponding bridge fungible token contract defined on the Fuel chain. These contracts handle messaging with the base layer gateway contract and are responsible for minting and burning the Fuel side token proxies.
 
-## Dependencies
```

Comment:
Why was this removed? Shouldn't there be some easy reference for developers? Especially since to run `forc-deploy` you need both of these dependencies.